### DISCUSSION TITLE
React Router 7 version?

### DISCUSSION THREAD

**Discussion by JasonColeyNZ** [sergiodxa/remix-utils#418] [General]
Hi there,

will there be a react router 7 version of these utils, I have just been working on upgrading an app over to RR7 and this dependancy has been widely used in my app?

**Comment by discoverlance-com** [comment]
I was thinking the same and if not, cloning this repository and updating it to create a new one to support the react router. But I'm wondering if this package will actually start to support it then there will be no need for such an effort.

**Comment by sergiodxa** [comment]
https://github.com/sergiodxa/remix-utils/discussions/417#discussioncomment-11416439
